test/utils.js

```diff
@@ -1,3 +1,6 @@
+const faker = require('faker');
+const { factory } = require('factory-girl');
+const { User } = require('../app/models');
 exports.user = {
```

Comment:
shouldn't you delete that? since you are defining the user with factory now

Reply:
I'm still using it, if i need an specific user data, i will use this object, i use the factory for random user creation